### ISSUE TITLE
chrome.MobileEmulation.DeviceMetrics should be a pointer

### ISSUE THREAD

**Issue by robertogden** [tebeka/selenium#101]
[This line](https://github.com/tebeka/selenium/blob/a7e71178aa40d9b298d464b912c4e831b2be4455/chrome/capabilities.go#L78) should be a pointer to DeviceMetrics so that it is omitted when empty during JSON serialization.

Otherwise this happens: 
```
{
  "mobileEmulation": {
    "deviceName": "Nexus 5",
    "deviceMetrics": {
      "width":0,
      "height":0,
      "pixelRatio":0
    }
  }
}
```

Thanks!

**Comment by minusnine** [comment]
I just merged PR #96, so this should be fixed.

Please re-pull and re-open this issue if any issues continue.

**Comment by robertogden** [comment]
Thanks!
